### PR TITLE
Deploy docs

### PR DESCRIPTION
**Pull request by runkecheng**

### CODE REVIEW

File: docs/KubeSphere/deploy_xenondb_on_kubesphere.md [hunk 1]
```diff
@@ -42,23 +42,26 @@ XenonDB 是基于 MySQL 的开源、高可用、云原生集群解决方案。
 2. 执行如下命令，将 XenonDB Chart 克隆到 KubeSphere 客户端节点中。
 
 ```bash
-git clone https://github.com/andyli029/xenondb-helm.git
+git clone https://github.com/radondb/xenondb-chart
```
Comment (andyli029) [inline docs/KubeSphere/deploy_xenondb_on_kubesphere.md:45]: ->
git clone https://github.com/radondb/xenondb

File: charts/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
-name: krypton
+name: xenondb
 description: Fast, reliable, scalable, and easy to use open-source relational database system.
-home: https://github.com/radondb/krypton
+home: https://github.com/radondb/xenondb-chart
```
Comment (andyli029) [inline charts/Chart.yaml:4]: ->
git clone https://github.com/radondb/xenondb

File: charts/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Introduction
 
-This chart bootstraps a single master and multiple slave [MySQL](https://MySQL.org) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+This chart bootstraps a single leader and multiple follower [MySQL](https://MySQL.org) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
```
Comment (hustjieke) [inline charts/README.md:4]:
```
## For Helm v2
$ cd charts
$ helm install . --name my-release
```
befor `cd charts`, we should do `git clone` opreration.

and 
```
helm install . --name my-release
```
should be 
```
helm install .  my-release
```

File: charts/templates/_helpers.tpl [hunk 4]
```diff
@@ -9,7 +9,7 @@ Expand the name of the chart.
 {{/*
 Create a default fully qualified app name.
 The format of host is "fullname"-0."fullname"."namespace" (podname.servicename.namespace), 
-and the MySQL limits the total length of master_host to 60 byte, 
+and the MySQL limits the total length of leader_host to 60 byte,
```
Comment (zhyass) [inline charts/templates/_helpers.tpl:12]: 改回master_host
mysql中没有leader_host 只有master_host

File: charts/templates/statefulset.yaml [hunk 5]
```diff
@@ -215,14 +215,14 @@ spec:
         - name: LEADER_STOP_CMD
           value: "/scripts/leader-stop.sh"
         {{- if .Values.mysql.initTokudb }}
-        - name: Master_SysVars
+        - name: Leader_SysVars
```
Comment (zhyass) [inline charts/templates/statefulset.yaml:218]: 此处的`Master_SysVars`、`Slave_SysVars` 应和 https://github.com/andyli029/krypton-helm/blob/master/dockerfile/xenon/xenon-entry.sh#L80 保持一致，如果要修改，请一并修改，并重新制作推送xenon镜像

File: docs/KubeSphere/deploy_xenondb_on_kubesphere.md (hunk 1 above)
Comment (zhyass) [inline docs/KubeSphere/deploy_xenondb_on_kubesphere.md:45]: git clone https://github.com/radondb/xenondb-chart.git

File: docs/KubeSphere/deploy_xenondb_on_kubesphere.md [hunk 6]
```diff
@@ -103,13 +106,13 @@ git clone https://github.com/andyli029/xenondb-helm.git
 拉取 XenonDB Chart。
 
 ```bash
-git clone https://github.com/andyli029/xenondb-helm.git
+git clone https://github.com/radondb/xenondb-chart
```
Comment (zhyass) [inline docs/KubeSphere/deploy_xenondb_on_kubesphere.md:109]: 同上

File: docs/Kubernetes/deploy_xenondb_on_kubernetes.md [hunk 7]
```diff
@@ -17,22 +17,23 @@ XenonDB 是基于 MySQL 的开源、高可用、云原生集群解决方案。
 执行如下命令，将 XenonDB Chart 克隆到 Kubernetes 中。
 
 ```bash
-git clone https://github.com/andyli029/xenondb-helm.git
+git clone https://github.com/radondb/xenondb-chart
```
Comment (zhyass) [inline docs/Kubernetes/deploy_xenondb_on_kubernetes.md:20]: 同上

**Review by zhyass — CHANGES_REQUESTED** [review]

**Comment by andyli029** [comment]
https://github.com/radondb/xenondb-chart
->
https://github.com/radondb/xenondb

**Pull request merged**